src/web/components/ArticleTitle.tsx

```diff
@@ -15,7 +15,11 @@ const sectionStyles = css`
 `;
 
 type Props = {
-    CAPI: CAPIType;
+    tags: CAPIType['tags'];
+    sectionLabel: CAPIType['sectionLabel'];
+    sectionUrl: CAPIType['sectionUrl'];
+    guardianBaseURL: CAPIType['guardianBaseURL'];
+    pillar: CAPIType['pillar'];
```

Comment:
This is great, thanks for doing this. I'm not sure about the CAPIType['etc'] syntax though. For 2 reasons

1. I've never seen it so I'm like, ummm, okay...
2. It hides the actual type, I have to dig into index to see what they are

Can we:

```
{
    tags: TagType[];
    sectionLabel: string;
    sectionUrl: string;
    guardianBaseURL: string;
    pillar: Pillar;
}
```

I realise that's repetition and if we change the type of `sectionUrl` in `CAPIType` we'd need to also update this too but that's true for most of the props we pass and I like the trade off of easier development/readability vs. harder updates.

Reply:
I think it's interesting - there's a case for 'component shouldn't know about parent' but then it feels weird to be repeating the types.

This autocompletes and does resolve to the actual type:

![2020-02-05 08 44 50](https://user-images.githubusercontent.com/638051/73825548-10f05680-47f4-11ea-9db4-520c6a3b0a30.gif)

However, maybe it makes more sense to *not* inherit from CAPIType but on the more important types (like we have for Pillar), we need to create a type for it. I'm happy to go with that route rather than resolve from CAPIType directly 👍